src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -7,8 +7,11 @@
 use DateTime;
 use DateTimeZone;
 use Doctrine\DBAL\Connection;
-use Doctrine\DBAL\DBALException;
-use Doctrine\DBAL\Platforms\PostgreSqlPlatform;
+use Doctrine\DBAL\Driver\PDO\Connection as PDOConnection;
+use Doctrine\DBAL\Exception as DBALException;
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
```

Comment:
it seems that dbal 2 provides only the MySqlPlatform with lower case. i think we should define an alias if MySQLPlatform does not exist to avoid regressions with dbal 2.